pkg/datagatherer/versionchecker/versionchecker.go

```diff
@@ -53,7 +54,9 @@ const (
 type Config struct {
 	// the version checker dg will also gather pods and so has the same options
 	// as the dynamic datagatherer
-	Dynamic                     k8s.ConfigDynamic
+	Dynamic k8s.ConfigDynamic
```

Comment:
I think it makes sense to have 2, one for nodes, and one for pods. However, I'd rename the pods one to something like `DynamicPod`